assets/sass/components/_buttons.scss

```diff
@@ -327,6 +327,27 @@ $buttonColor: get-color('brand', 'primary');
     }
 }
 
+/**
+ * Button group separated left
+ * Allows a set of buttons to be split from other buttons
+ * with a divider on their left
+ */
+.o-button-group--separated {
```

Comment:
Can this be rewritten so that it's not a modifier? Asking based on the others (`o-button-group-flex` etc.) being standalone classes. It might not make sense here.